code/game/objects/items.dm

```diff
@@ -299,6 +304,10 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 		if(resistance_flags & FIRE_PROOF)
 			. += "[src] is made of fire-retardant materials."
 
+	/// Code for showing offensive statistics via a hyperlinked tag
+	if(force >= 5 || throwforce >= 5 || override_notes || offensive_notes) /// Only show this tag for items that could feasibly be weapons, shields, or those that have special notes
+		. += "<span class='notice'>It appears to have an ever-updating holographic <a href='?src=[REF(src)];examine=s'>warning label.</a></span>"
```

Comment:
I don't really like new Topic() handles, I feel like there should be a better way. Whether there actually is one, I don't know.

Have you considered putting this stuff into "examine more" instead?

Reply:
examine more has terrible UI and noone knows it exists.